src/components/dashboard/DashboardPage.jsx

```diff
@@ -11,39 +11,23 @@ import { LoadingSpinner } from '../loading-spinner';
 import { DashboardMainContent } from './main-content';
 import { DashboardSidebar } from './sidebar';
 import Hero from './Hero';
+import NotFoundPage from '../NotFoundPage';
 
 import { fetchEntepriseCustomerConfig } from './data/service';
 
 const DashboardPage = (props) => {
-  const initialPageContext = {
```

Comment:
FYI, the logic for fetching an enterprise config is moving to a custom hook within the `EnterprisePage` component so that it can be more easily shared across multiple pages. See https://github.com/edx/frontend-app-learner-portal-enterprise/pull/18/files#diff-6fc9b7897f8266d914ac40ef20b0550dR23.

I think how you have it here in this PR is fine for now. Once it's merged, I will make sure to rebase the course detail page branch with master to bring in this PR's changes and update my branch as needed.

Reply:
@bbaker6225 Update: I split the custom hook logic mentioned above into a smaller PR that can be shipped on its own. Take a look at https://github.com/edx/frontend-app-learner-portal-enterprise/pull/19.

We can either get #19 reviewed/merged and update this PR accordingly or vice versa. Up to you!

Reply:
Ah just updated this PR. I'll take a look at #19 now